tools/pubsys/src/aws/publish_ami/mod.rs

```diff
@@ -413,6 +421,8 @@ pub(crate) async fn modify_regional_snapshots(
 pub(crate) async fn modify_image(
```

Comment:
Adding the args does the job and is the pragmatic solution for sure.  

But given that this function now has 7 arguments, perhaps we should look into some sort of config struct or builder to clean this up.  I'd probably lean towards a config struct in this case to avoid the overhead of a builder at least for now.

Reply:
Yeah I could see this being a good thing:

```rust
struct ModifyOptions {
    user_ids: Option<Vec<String>>,
    user_groups: Option<Vec<String>>,
    organization_arns: Option<Vec<String>>,
    organization_unit_arns: Option<Vec<String>>,
}
```

Reply:
^ Looks good; I'd probably shorten `organization_` down to `org_`.